docs/checkouts/lifecycle.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Checkout lifecycle
+sidebar_label: Lifecycle
+description: Understand a Checkout lifecycle
+keywords:
+  - checkouts
+  - features
+  - lifecycle
+---
+
+import Figure from "@site/src/components/Figures";
+import CheckoutStatuses from "@site/static/img/checkout_statuses.png";
+
+# Lifecycle
+
+## Overview
+
+A Checkout is not a Payment; it has its own data model, lifecycle, and statuses.
+
+### Diagram
+
+- When a Checkout is created it status is set to `created`.
+- If a payment attempt fails, it transitions to the status `attempted`
+  and remains there until it either expires or a payment is successful.
+- If a payment succeeds, it transitions to the status `completed`.
+- If not completed within 1 week, it automatically transitions to the status `expired`.
```

Comment:
I'd add one more point here:
'A checkout can have multiple failed payment attempts and/or 1 succeeded payment attempt - see Checkout Completion Attribute' (link to that section below if possible)